src/main/java/io/cosmosoftware/kite/util/WebDriverUtils.java

```diff
@@ -383,6 +383,16 @@ public static boolean isElectron(WebDriver webDriver) {
     Capabilities capabilities = ((RemoteWebDriver) webDriver).getCapabilities();
     return "chrome".equals(capabilities.getBrowserName()) && capabilities.getVersion().isEmpty();
   }
+
+  /**
+   * @return true if it is chrome
+   */
+  public static boolean isChrome(WebDriver webDriver) {
+    Capabilities capabilities = ((RemoteWebDriver) webDriver).getCapabilities();
+    return "chrome".equals(capabilities.getBrowserName());
```

Comment:
this is also true for eclectron based app

Reply:
it's fine. what I want to do on Chrome should also work on electron (i guess)